tools/typescript2fable.js

```diff
@@ -310,9 +335,12 @@ function printModule(prefix) {
     return function(mod) {
         var template = prefix + templates.module
             .replace("[NAME]", escapeKeyword(mod.name));
-            
-        template = append(template, mod.interfaces.map(
-            printInterface(prefix + "    ", mod.name)).join("\n\n"));
+
+        var interfaces = mod.interfaces && mod.interfaces.length > 0
+                            ? mod.interfaces.map(printInterface(prefix + "    ", mod.name)).join("\n\n")
+                            : "";
```

Comment:
Not sure to be fair, when I was playing with this code, I've expected exception here... but that could be result of me messing something in other parts of code.